File: PRAK304-2310817210001-AndraBraputraAkbarSaleh.c
#include <stdio.h>

int main() {
    int a;
    scanf("%d", &a);
    if (a > 0 && a < 10) {
        printf("satuan");
    } else if (a >= 10 && a < 20) {
        printf("belasan");
    } else if (a > 20 && a < 100) {
        printf("puluhan");
    } else if (a == 0) {
        printf("nol");
    } else if (a >= 100) {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    return 0;
}

File: PRAK304-2310817210001-AndraBraputraAkbarSaleh.py
a = int(input())
if 10 > a > 0:
    print("satuan")
elif 10 <= a < 20:
    print("belasan")
elif a == 0:
    print("nol")
elif 20 <= a < 100:
    print("puluhan")
elif a >= 100:
    print("Anda Menginput Melebihi Limit Bilangan")